### PR TITLE
[ftpupload] Add configurable passive port

### PR DESCRIPTION
**Pull request by schinken**
The PR adds the functionality to optionally configure the passive FTP port range according to org.apache.ftpserver (https://mina.apache.org/ftpserver-project/configuration_passive_ports.html).

This is useful when you want to reduce the attack surface and reduce the number of ports to be allowed in your firewall.

Be aware:
I tried several hours to setup the dev environment with intellij (yes, I followed the docs ;)) - but all I got was building a single binding (this one) with mvn in the console. Testing was also hard because openhab debug always pulled the online version of the binding instead my version from the addon directory. 

So for testing there is help needed. Thanks in advance!

### CODE REVIEW

File: bundles/org.openhab.binding.ftpupload/src/main/java/org/openhab/binding/ftpupload/internal/FtpUploadHandlerFactory.java [hunk 1]
```diff
@@ -116,9 +118,21 @@ protected synchronized void modified(ComponentContext componentContext) {
             }
         }
 
+        if (properties.get("passivePorts") != null) {
+            String strPassivePorts = properties.get("passivePorts").toString();
+            if (StringUtils.isNotEmpty(strPassivePorts)) {
```
Comment (Hilbrand) [inline bundles/org.openhab.binding.ftpupload/src/main/java/org/openhab/binding/ftpupload/internal/FtpUploadHandlerFactory.java:123]: Can you change this to use standard Java. We're trying to get ride of apache libraries, and even while it's used on other places here, now new usages should be added.
```suggestion
            if (!strPassivePorts.isEmpty()) {
```

**Comment by schinken** [comment]
@Hilbrand I applied your suggestion. Thanks for the feedback!

**Comment by Hilbrand** [comment]
Look like you need to run `mvn spotless:apply` on your binding to fix the build issues.

**Comment by schinken** [comment]
Ok, done!

**Review by cpmeister — APPROVED** [review]
LGTM

**Comment by cpmeister** [comment]
Can you please update the readme as well?

**Pull request merged**